### ISSUE TITLE
Add how to jump start a car

### ISSUE THREAD

**Issue by patepelo** [ligi/SurvivalManual#268]
Here an example:

![image](https://github.com/ligi/SurvivalManual/assets/23003127/de8d9841-7303-43bb-aa5e-acaaac1d3c1d)

Other ideas, how to change a tire.

While it may seem obvius, for many it isnt and it can happen in a bad situation without mobile coverage.

**Comment by ligi** [comment]
Great idea! Can you add it to the wiki? Just make sure the image used is public domain

**Comment by patepelo** [comment]
New to this project, do you want me to edit the wiki to write the steps?
 
https://en.wikibooks.org/wiki/Automobile_Repair/Jump_start

This seems like a good reference but maybe too long. I'll have to check if the wikibooks licence compatible is or if we could include in the license in the app.

**Comment by ligi** [comment]
yeah - the content of our wiki here is basically baked in the app in build.
This looks good but long - license seems good (cc) when attributed.
